### DISCUSSION TITLE
10/24 조지연 블로그 생성

### DISCUSSION THREAD

**Discussion by oratio100** [Orm-camp/blog-feedback#20] [동글동글 3기 피드백]
https://velog.io/@nemonemo

저는 velog에서 블로그를 생성했습니다.
처음에는 배우고 있는 개념을 정리하는 글들을 쓸 것 같고,
이후에는 오류 발생 기록이나 해결방법을 기록하는 용도로 글을 쓰려고 합니다.

**Comment by ghost** [comment]
지윤환: 블로그 생성하시느라 고생 많으셨습니다. 
앞으로도 화이팅입니다!

**Comment by kimjigyun** [comment]
네모를 좋아하시나봐요 ㅎㅎ 블로그 글 화이팅 입니다 !
